AcademicReward/Views/PurchaseHistoryPage.xaml.cs

```diff
@@ -1,19 +1,30 @@
+using AcademicReward.Database;
+using AcademicReward.ModelClass;
 using System.Collections.ObjectModel;
 
 namespace AcademicReward.Views;
 
 /// <summary>
-/// PurchaseHistoryPage shows all purchased items for a profile
```

Comment:
Could we keep these comments? I'm sure you didn't mean to get rid of them, unless...did you?

Reply:
I think this was a merge conflict I must've removed by mistake, I'll add it back